### PR TITLE
Tree for tractometry

### PR DESCRIPTION
**Pull request by mdesco**
Tree for tractometry getting rbx bundles, tractoflow metrics and then fw and noddi metrics in 2 separate scripts.

Open to suggestions.

### CODE REVIEW

File: tree_for_tractometry.sh [hunk 1]
```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+usage() { echo "$(basename $0) [-r RBX] [-t tractoflow/results] [-o output]" 1>&2; exit 1; }
```
Comment (GuillaumeTh) [inline tree_for_tractometry.sh:2]: I think we could have noddi and fw here too as an optionnal argument. In the for loop we could check if the argument -n or -f is empty or not. What do you think @mdesco ?

Comment (mdesco) [reply tree_for_tractometry.sh:2]: sure but my bash skills are too weak to attempt this... go for it if you can.

Comment (GuillaumeTh) [reply tree_for_tractometry.sh:2]: Ok I will do it based on you scripts.

**Comment by mdesco** [comment]
Before merging this. Can you add a warning to the use that we recommend < 100 subjects per tractometry batch. 

I'm currently testing on beluga and 100 subjects is about the max not to explode the 1M file quota. thanks

**Comment by GuillaumeTh** [comment]
Replace by PR #6

**Pull request closed without merging**